datus/storage/document/nav_resolver/hugo_resolver.py

```
@@ -0,0 +1,126 @@
+
+
+class HugoResolver(BaseNavResolver):
+    """Resolves nav_path from Hugo directory structure and frontmatter.
+
+    Requires ``extra_context`` containing a dict mapping file paths to their
+    frontmatter metadata (extracted from already-fetched documents).
+    """
+
+    def resolve(
+        self,
+        config_content: str,
```

Ruff (0.14.14)

[warning] Unused method argument: `config_content`

(ARG002)